Common/Product/SharedProject/ProcessOutput.cs

```diff
@@ -184,23 +184,20 @@ sealed class ProcessOutput : IDisposable {
                     redirector,
                     quoteArgs,
                     outputEncoding,
-                    errorEncoding
-                );
-            }
-
-            var psi = new ProcessStartInfo(filename);
-            if (quoteArgs) {
-                psi.Arguments = string.Join(" ",
-                    arguments.Where(a => a != null).Select(QuoteSingleArgument));
-            } else {
-                psi.Arguments = string.Join(" ", arguments.Where(a => a != null));
-            }
-            psi.WorkingDirectory = workingDirectory;
-            psi.CreateNoWindow = !visible;
-            psi.UseShellExecute = false;
-            psi.RedirectStandardError = !visible || (redirector != null);
-            psi.RedirectStandardOutput = !visible || (redirector != null);
-            psi.RedirectStandardInput = !visible;
+                    errorEncoding);
+            }
+
+            var psi = new ProcessStartInfo("cmd.exe") { 
+                Arguments = string.Format(@"/S /C pushd {0} & {1} {2}",
```

Comment:
Yes, between 45 and 120 seconds from what I can gather (it seem to differ by Windows version.) I don't see any easy way to adjust the timeout for a single command.

Not ideal, but also not a huge problem. It's more an external dependency that we should be aware of when trying to diagnose reported problems.